frontend/src/component/releases/ReleasePlanTemplate/MilestoneList.tsx

```diff
@@ -9,7 +12,10 @@ interface IMilestoneListProps {
     setMilestones: React.Dispatch<
         React.SetStateAction<IReleasePlanMilestonePayload[]>
     >;
-    setAddStrategyOpen: (open: boolean) => void;
+    openAddStrategyForm: (
+        index: number,
```

Comment:
I think this is going to be increasingly difficult to maintain. Maybe we should call it something else. I think this is the `milestoneIndex`, right?

Reply:
It's being replaced by milestoneId in a separate PR!